content/cybersecurity/concepts/nmap/terms/tcp-syn-scan/tcp-syn-scan.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'TCP SYN Scan'
+Description: 'Performs a scan quickly, efficientily and stealthly. The most popular scan option'
+Subjects:
+  - 'Networking'
+  - 'Cybersecurity'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'introduction-to-ethical-hacking'
+  - 'paths/scan-systems-with-nmap'
+---
+
+A **TCP SYN scan** runs by default when running Nmap as root or Administrator. It is the most popular scan option.
+Quick and efficient, this scan can indicate open, filtered and closed port states.
+It is requested by passing the -sS option to Nmap, but when running as root or Administrator, -sS is usually omitted.
+TCP SYN Scan is also stealthy because it never completes the full TCP connection, so is less likely to be blocked by firewalls.
```

Comment:
Can we place these on the same lines so they appear as a paragraph? In addition, let's wrap any shell snippets like `-sS` in backticks.